src/test/end-to-end-tests/test-runner-bot.ts

```diff
@@ -434,3 +446,13 @@ async function ensureVoiceConnection(): Promise<void> {
 
     await bot.connect();
 })();
+
+process.on("SIGINT", async () => {
+    console.log("Caught SIGINT, leaving voice channel");
+    if (voiceConnection) {
+        getVoiceChannel().leave();
+        await delay(1000);
```

Comment:
Why do we wait 1s here?

Reply:
`.leave()` is a synchronous function despite being an async action. Give some time for it to complete before we exit the process